src/popup/popup.html

```diff
@@ -204,6 +204,17 @@ <h6 class="vocab heading info__label margin-bottom-medium">Share</h6>
         </div>
       </div>
     </div>
+  <div class="modal">
```

Comment:
The indentation is still not right:
- The opening div should align with the above closing div tag.
- The closing div should align with the below opening footer tag.
- The indentation is of 2 spaces (mentioned under [editorconfig heading in INSTRUCTIONS.md](https://github.com/creativecommons/ccsearch-browser-extension/blob/master/INSTRUCTIONS.md#editorconfig)) but you have used 6 at most places, and 5 and 3 also.

Reply:
Sorry for the recurrent mistake, I will keep in mind from next time.